Podfile

```diff
@@ -0,0 +1,14 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '8.0'
+# Uncomment this line if you're using Swift
+use_frameworks!
+
+target 'WordPressComKit' do
+	pod 'Alamofire'
```

Comment:
If this is meant to be a framework that any third party could add to their project to interact with WordPress.com, I think having no external dependencies is a great goal to have. I understand Alamofire can get us to a first version faster, but long term can we use NSURLSession directly? What is Alamofire giving us that we'd need to implement ourselves?